include/common/oclengine.hpp

```diff
@@ -37,20 +37,30 @@ class OCLEngine {
     cl::CommandQueue* GetQueuePtr();
     /// Get a pointer to the Apply2x2 function kernel
     cl::Kernel* GetApply2x2Ptr();
+    /// Get a pointer to the Apply2x2Norm function kernel
+    cl::Kernel* GetApply2x2NormPtr();
+    /// Get a pointer to the ROL function kernel
```

Comment:
Comments needs updating in various places

Reply:
Done.